hepcrawl/testlib/fixtures.py

```diff
@@ -10,12 +10,13 @@
 from __future__ import absolute_import, division, print_function
 
 import os
+import json
 
 from scrapy.http import Request, TextResponse
 from scrapy.selector import Selector
 
 
-def fake_response_from_file(file_name, url='http://www.example.com', response_type=TextResponse):
+def fake_unit_response_from_file(file_name, url='http://www.example.com', response_type=TextResponse):
```

Comment:
I would add here the 'test suite' as parameter (what you call test type after), with a default value for unit.